faq/site_progress_snapshot_legend.php

```diff
@@ -31,14 +31,14 @@
 
 <p>In addition each activity has an icon that represents your current ability to work in that stage:
 <ul>
-  <li><div class='access-yes'>✓</div> - You have access to, and are able to work in, this stage.</li>
-  <li><div class='access-eligible'>ⓘ</div> - You satisfy the basic criteria and are eligible to work in this stage. Clicking on the icon will provide more information on how to obtain access.</li>
-  <li><div class='access-no'>✗</div> - You are not yet eligible to work in this stage. Clicking on the icon will show you the entrance requirements for that stage.</li>
+  <li><span class='access-yes' style='float: none;'>✓</span> - You have access to, and are able to work in, this stage.</li>
```

Comment:
Looks like access-* classes are only used in this faq and in the activity_hub:
https://github.com/DistributedProofreaders/dproofreaders/search?q=access-eligible
Should we instead just create a separate class to apply to the icons in activity_hub and avoid inline style here?

Reply:
The inline style, is, quite frankly, a bandaid. It's to override the "float right" that's used specifically for the snapshot table, so that the icons appear in their proper places in the list. My original plan was to update the text in this file, and the beginner help text on the Activity Hub, but the more I dug into it, the more I realized that would be overwhelming to beginners. After consulting with @lhamilton1 and @cpeel, I think that we've decided to create a new wiki page in the DP official documentation that describes the Snapshot Table on the Activity Hub and to remove this file altogether (and the beginner help text on the AH).

But the removal needs to wait until the wiki page is created -- so, another MR. I could have just left the file alone altogether, but am picky enough that I wanted the text to be correct and the icons to display properly, even if it's slated for the chopping block. 😕 

So, at this point, no, I don't think we need to create a separate class at this point (that may change, though).

Reply:
Agreed that doing this inline is probably the best bandaid right now given the future direction.